go/obscuronode/host/p2p/p2p.go

```diff
@@ -165,12 +168,12 @@ func (p *p2pImpl) broadcast(msgType Type, bytes []byte) {
 	}
 
 	for _, address := range p.PeerAddresses {
-		sendBytes(address, msgEncoded)
+		p.SendBytes(address, msgEncoded)
 	}
 }
 
-// Sends the bytes over P2P to the given address.
-func sendBytes(address string, tx []byte) {
+// SendBytes Sends the bytes over P2P to the given address.
+func (p *p2pImpl) SendBytes(address string, tx []byte) {
```

Comment:
I can see why you're doing this (to avoid duplicating code in `l2_network.go`, but personally I'd rather see code duplication than exposing a public method on this interface just for simulation purposes.

Reply:
I might have to duplicate the p2p-local chan logic send in that case too, which might be troublesome.